docker-compose.yml

```diff
@@ -61,7 +61,7 @@ services:
     command: "luigid"
 
   bigmetadata:
-    image: carto/bigmetadata
+    image: carto/bigmetadata:1.1
```

Comment:
are we planning to upload to Docker Hub the new version and keep it there? If not we could just generate the new image with the existing tag and upload it when this PR is merged

Reply:
In fact, I already pushed it so it was available to staging. Shouldn't I have done it?

Reply:
yes, no problem, but we use to update the `carto/bigmetadata` image instead of creating new versions (anyway we can do it later, so no problem)

Reply:
While we can do it later, I should revert the versioning. In general, versioning is a good practice. Why would you not want to use it here? If it's not changed, shouldn't the downloaded images be removed?

Reply:
100% agree that versioning is a good practice, I was just thinking out loud because we were not using versions until now (this doesn't mean that it was the best practice, but I guess that the reason is to avoid versioning trees).
Anyway, adding a new version of the image is something that we don't do _that often_, so I would keep the versioning (and even enforce versioning in future cases)